Northstar.Custom/mod.json

```
@@ -39,7 +39,7 @@
 		},
 		{
 			"Path": "weapons/mp_weapon_peacekraber.nut",
-			"RunOn": "( CLIENT || SERVER ) && MP"
+			"RunOn": "( CLIENT || SERVER )"
```

```suggestion
			"RunOn": "CLIENT || SERVER"
```

to keep style of the other context conditions